Server/Services/LeaderBoardService.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Server.Controllers;
+using Server.Models;
+
+namespace Server.Services
+{
+    public class LeaderBoardService
+    {
+        private readonly IConfiguration _configuration;
+        private readonly AppDbContext _context;
+        private readonly int _numberOfTopPlayers;
+        private readonly int _maximumDelta;
+        private readonly int _baseRating;
+        private readonly int _divisionPoints;
+
+        public LeaderBoardService(IConfiguration configuration, AppDbContext context)
+        {
+            _configuration = configuration;
+            _context = context;
+            _numberOfTopPlayers = Int32.Parse(configuration["NumberTopPlayers"]);
+            _maximumDelta = Int32.Parse(configuration["MaximumDelta"]);
+            _baseRating = Int32.Parse(configuration["BaseRating"]);
+            _divisionPoints = Int32.Parse(configuration["DivisionPoints"]);
+        }
+
+        /// <summary>
+        /// Should return the top 100 top players. If you need to change that, change the constant at the beginning
```

Comment:
I don't see much value in that doc. The count of players shouldn't be important for the consumer, and the note about changing is an implementation detail